config/routes.rb

```diff
@@ -23,4 +23,8 @@
   namespace :admin do
     get '/dashboard', to: "users#show"
   end
+
+  scope :dashboard do
+    get 'items', to: "merchants#items_index", as: :merchant_items_index
```

Comment:
Hm... This page should only be visible to merchants, right? Seems like this may be a good time to add a `Merchant::BaseController` & `merchant` module. Then we would just have `resources :items, only: [:index]` in there (going to `merchants/ItemsController#index`) instead of  `merchants#items_index`. We can do it together if you'll be in today (or maybe screen share over Slack if not).